CruCentralCoast/Community/MinistryTeams/MinistryTeamDetailsVC.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  MinistryTeamDetailsVC.swift
+//  CruCentralCoast
+//
+//  Created by Landon Gerrits on 4/16/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+
+class MinistryTeamDetailsVC: UIViewController {
+    
+    @IBOutlet weak var bannerImageView: UIImageView!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var movementLabel: UILabel!
+    @IBOutlet weak var leaderNamesLabel: UILabel!
+    @IBOutlet weak var summaryLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.navigationItem.largeTitleDisplayMode = .never
+    }
+
+    @IBAction func joinMinistryTeam() {
+        // TODO
+        self.presentAlert(title: "Join Ministry Team", message: "Coming Soon...")
+    }
+    
+    func configure(with ministryTeam: MinistryTeam) {
+        DispatchQueue.main.async {
+            self.nameLabel.text = ministryTeam.name
+            self.movementLabel.text = ministryTeam.movement?.name
+            self.leaderNamesLabel.text = "Leaders: \(ministryTeam.leaderNames ?? "N/A")"
+            self.summaryLabel.text = ministryTeam.summary
+            // Fetch the image from local storage or download it
+            if let imageLink = ministryTeam.imageLink {
```

Comment:
just out of curiosity, this code seems to be repeated each time an image is used or downloaded. Is there any way this could be more generic? I know it is already using an added extension, but it is just an idea.

Reply:
Actually yes! UIImageView extensions could be used here. It would not work for cells since they get reused, but since the detail view will never be reused, I could use the extension to reduce these 7 lines to a single line. I will have to make a slight update to one of the UIImageView extension methods once the other PR gets merged